src/coreclr/vm/interpexec.cpp

```diff
@@ -7,11 +7,12 @@
 #include "gcenv.h"
 #include "interpexec.h"
 #include "callstubgenerator.h"
+#include "frames.h"
 
 // for numeric_limits
 #include <limits>
 
-void InvokeCompiledMethod(MethodDesc *pMD, int8_t *pArgs, int8_t *pRet)
+void InvokeCompiledMethod(MethodDesc *pMD, int8_t *pArgs, int8_t *pRet, PCODE overrideTarget = (PCODE)0)
```

Comment:
I would just always pass in the target address. The place where we have been calling this method before would just pass in the pMD->GetMultiCallableAddrOfCode(CORINFO_ACCESS_ANY).